### PR TITLE
Implement PDF Export Generation

### PR DESCRIPTION
**Pull request by nayancprakash**
# Implemented Features
* implements PDF generation for both owners and participants
* uses uploaded background PDF as template for generated PDF

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@
   create_table 'diffs', force: :cascade do |t|
     t.integer 'layer_id', null: false
     t.integer 'seq'
-    t.json 'data', default: '{}', null: false
+    t.json 'data', default: '""{}""', null: false
```
Comment (calebschoepp) [inline db/schema.rb]: IDK why this happens but we don't want this change.

File: db/migrate/20220310153625_create_exports.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@ def change
       t.references :notebook, null: false, foreign_key: true
       t.references :user, null: false, foreign_key: true
       t.boolean :ready
+      t.boolean :failed
```
Comment (calebschoepp) [inline db/migrate/20220310153625_create_exports.rb]: This should be done in a new migration like `rails g migration add_failed_to_exports failed:boolean`.

**Review by calebschoepp — CHANGES_REQUESTED** [review]
Couple comments but great job!

**Review by calebschoepp — APPROVED** [review]
LGTM! :crown:

**Review by ryankortbeek — APPROVED** [review]
LGTM great job!

**Pull request merged**